app/src/main/java/com/willowtree/vocable/room/PresetCategoryDeleted.kt

```diff
@@ -0,0 +1,8 @@
+package com.willowtree.vocable.room
+
+import androidx.room.ColumnInfo
+
+data class PresetCategoryDeleted(
+    @ColumnInfo(name = "category_id") val categoryId: String,
+    @ColumnInfo(name = "deleted") val deleted: Boolean
+)
```

Comment:
I had not realized you could interface with Room like this 🤯 I would've just written another SQL statement

Reply:
this is so good!!!

Reply:
Yea this is pretty poorly document, but there's some info here: https://developer.android.com/reference/androidx/room/Update !